odk/src/main/java/org/motechproject/odk/service/factory/FormDefinitionImportServiceFactory.java

```diff
@@ -0,0 +1,50 @@
+package org.motechproject.odk.service.factory;
+
+import org.motechproject.odk.domain.ConfigurationType;
+import org.motechproject.odk.service.FormDefinitionImportService;
+import org.motechproject.odk.service.impl.FormDefinitionImportServiceKobo;
+import org.motechproject.odk.service.impl.FormDefinitionImportServiceODK;
+import org.motechproject.odk.service.impl.FormDefinitionImportServiceOna;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * Factory class for {@link FormDefinitionImportService}
+ */
+
+@Component
+public class FormDefinitionImportServiceFactory {
+
+    @Autowired
+    private FormDefinitionImportServiceODK formDefinitionImportServiceODK;
+
+    @Autowired
+    private FormDefinitionImportServiceOna formDefinitionImportServiceOna;
+
+    @Autowired
+    private FormDefinitionImportServiceKobo formDefinitionImportServiceKobo;
+
+
+    /**
+     * Returns the appropriate {@link FormDefinitionImportService} bean for the configuration type.
+     * @param type The {@link ConfigurationType} associated with the form definition.
+     * @return {@link FormDefinitionImportService}
+     */
+    public FormDefinitionImportService getService(ConfigurationType type) {
+
+        switch (type) {
+            case ODK:
+                return formDefinitionImportServiceODK;
+
+            case ONA:
+                return formDefinitionImportServiceOna;
+
+            case KOBO:
+                return formDefinitionImportServiceKobo;
+
+            default:
+                return null;
```

Comment:
Another case of a suspect null return. Does it make sense to return null? If this means we fail on the next few lines, better throw some sort of a configuration or IllegalStateException here